packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -175,8 +175,8 @@ library ChainAssertions {
         // Check that the contract is initialized
         assertInitializedSlotIsSet({ _contractAddress: address(messenger), _slot: 0, _offset: 20 });
 
-        require(address(messenger.OTHER_MESSENGER()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "CHECK-L1XDM-20");
-        require(address(messenger.otherMessenger()) == Predeploys.L2_CROSS_DOMAIN_MESSENGER, "CHECK-L1XDM-30");
+        require(address(messenger.OTHER_MESSENGER()) == address(0), "CHECK-L1XDM-20");
```

Comment:
I was confused why this didn't fail but it's because we no longer use this check when deploying the proxies. Can we still put this behind the if/else so that it does the correct check depending on if it's a proxy or not?

Reply:
See your change to L1StandardBridge as an example of what to do

Reply:
fixed